mavros/src/plugins/local_position.cpp

```diff
@@ -91,16 +97,14 @@ class LocalPositionPlugin : public MavRosPlugin {
 				pos_ned.x, pos_ned.y, pos_ned.z,
 				pos_ned.vx, pos_ned.vy, pos_ned.vz);
 
-		/* TODO: check convertion to ENU
-		 * I think XZY is not body-fixed, but orientation does.
-		 * Perhaps this adds additional errorprone to us.
-		 * Need more tests. Issue #49.
-		 *
-		 * orientation in ENU, body-fixed
-		 */
 		tf::Transform transform;
 		transform.setOrigin(tf::Vector3(pos_ned.y, pos_ned.x, -pos_ned.z));
-		transform.setRotation(uas->get_attitude_orientation());
+		geometry_msgs::QuaternionStampedPtr q = boost::make_shared<geometry_msgs::QuaternionStamped>();
```

Comment:
It is required to pass pointer somewhere (nh.publish)? If not use local variable.